src/containers/tutor-home-page/general-info-step/GeneralInfoStep.jsx

```diff
@@ -42,7 +46,15 @@ const GeneralInfoStep = ({ btnsBox }) => {
   const handleCityChange = (value) => {
     setCity(value)
   }
-
+  useEffect(() => {
+    handleStepData(stepLabel, {
+      firstName,
+      lastName,
+      country,
+      city,
+      professionalSummary
+    })
+  }, [firstName, lastName, country, city, professionalSummary])
```

Comment:
Please pay attention to the linter warning here.

Reply:
@vldkk but the error still hasn't gone away